### ISSUE TITLE
Top 10 Addons showing less then 10 addons in Germany and Russia

### ISSUE THREAD

**Issue by SuYoungHong** [mozilla/Fx_Usage_Report#51]
@openjck brought up the issue that in certain countries (DE, RU), top10addons is only showing 9 addons. 

currently investigating

**Comment by SuYoungHong** [comment]
Addons are identified by the combination of `addon.id` and `addon.name`. The `top_10_addons_on_date` is identifying the top10 addons by `addon.id` and `addon.name`, 

<img width="664" alt="screen shot 2018-05-21 at 5 51 48 pm" src="https://user-images.githubusercontent.com/7693575/40374384-54091c48-5dbf-11e8-99fe-721db0147bc1.png">

However, when two addons have the same name, (for example, WEB.DE.mailcheck above), one overwrites the other when converted to dict/json, since we are defining addons in the report by their name only (and not id/name), resulting in 9 (less then 10) 'top addons'. 

After consulting with @benmiroglio offline, we decided to keep only 1 addon id/name combination per name.

**Comment by openjck** [comment]
Any update on this?

**Comment by SuYoungHong** [comment]
Yes, after our last meeting, we agreed to report both the addon name addon id (represented as a link to the page for that addon).

I'll need to update the pipeline to produce both addon name and id (instead of just addon name like it currently does). 

the shape of the data for this metric will change slightly... 

there should be some work required on the front end side required to include the link in the top addons table 

this will probably require a reference dataset of addon-ids and urls in the addons website... 

lets sync up in person to hammer out the details?

**Comment by openjck** [comment]
Sure. I don't know how helpful I can be on the data processing side of things, but I'd be happy to answer any questions.

**Comment by openjck** [comment]
Are the usage numbers for 9 add-ons that *are* listed inaccurate because of this issue?

**Comment by SuYoungHong** [comment]
@openjck , hey John, 

So we discussed in the last team meeting and we agreed that it's okay to leave this bug as it is, and we can fix it in the next interation. 

The 9 addons being shown currently are not inaccurate, whats happening is that there are 2 addons with the same name (but different ids) in the top 10. When we process the data, because it's a python dict, there can't be 2 key/value pairs with the same key, so the first addon with the same name is being overwritten by the second addon with the same name. 

So the addon being displayed is the second of the 2 addons with the same name, and the % of users associated with that second addon. 

So the data isn't wrong, just incomplete.

**Comment by openjck** [comment]
Just wanted to check in on this issue. Do we have an estimate as to when it can be fixed?
